src/redux/actions/cellSets/updateCellSetsClustering.js

```diff
@@ -11,6 +11,10 @@ const updateCellSetsClustering = (experimentId, resolution) => async (dispatch,
     loading, error,
   } = getState().cellSets;
 
+  const {
+    startDate: pipelineStartDate,
```

Comment:
I would call this qcStartDate (to begin the naming shift from pipeline to qc)

Reply:
changed to `qcStartDate`